client/src/utils/sortFunctions.ts

```diff
@@ -0,0 +1,6 @@
+import { Strat } from '@/api/models/Strat';
+
+export const sortDateAddedASC = (a: Strat, b: Strat) =>
+  new Date(b.createdAt).getTime() - new Date(a.createdAt).getTime();
+export const sortDateAddedDESC = (a: Strat, b: Strat) =>
+  new Date(a.createdAt).getTime() - new Date(b.createdAt).getTime();
```

Comment:
Shouldn't the filename be in kebab-case?

Reply:
depends on who you ask :P For now, all the util files are in camelCase, maybe I might change them all in bulk later.